pyproject.toml

```diff
@@ -77,8 +77,5 @@ select = [
     "W",
 ]
 
-[tool.ruff.isort]
-force-single-line = true
```

Comment:
Why remove this? Personally I'm a big fan of placing imports on single lines - it makes for easier to read git diffs and reduces the number of merge conflicts (see https://github.com/asottile/reorder-python-imports#why-this-style)

Reply:
Most of the project actually failed this rule (not sure how that happened), so I removed it to make the autofix diff smaller (and to deviate less from ruff defaults). 

The merge conflict point is fair, but on the other hand it can lead to much longer boilerplatey import sections.

Can add a new PR to turn it back on though!